tests/training/test_transformation.py

```diff
@@ -5,27 +5,36 @@
 from numpy import nan, round, sqrt, floor, log as ln
```

Comment:
🙈  it is hard to track what happened with this file, from what I understood:
- Some code style
- Add new tests to existing ones covering the use cases with prefix/suffix on transformers